www/README.md

```diff
@@ -1,3 +1,47 @@
-# CompE+ www
```

Comment:
Maybe not for this PR but at a high level is this how we want to structure the README b/c I'd like to mirror it in the api. IMO it is good to have a guide to each script but I also think there should be general stuff about setting up dev environment, local testing, etc.

Reply:
When you say mirror api, do you mean like the headers and general content will be similat to api? Then we'd have another markdown for setting up dev env, local testing, etc.?

Reply:
I'm saying I would like the structure of the api and www readmes to be similar.

I think as an example they should have sections like setup, local development, testing, code style, scripts, etc.

Reply:
Makes sense. I'll put this in the notion as part of the technical design section so we can keep it in mind.